Libraries/Animated/src/createAnimatedComponent.js

```diff
@@ -17,7 +17,15 @@ const AnimatedProps = require('./nodes/AnimatedProps');
 const React = require('React');
 const ViewStylePropTypes = require('ViewStylePropTypes');
 
+const invariant = require('fbjs/lib/invariant');
+
 function createAnimatedComponent(Component: any): any {
+  invariant(
+    Component.prototype && Component.prototype.isReactComponent,
+    '`createAnimatedComponent` does not support stateless functional components, ' +
```

Comment:
tiny nit: "," should be a ";" I think

Reply:
I'll trust you on that one. I never know when `;` should be used, I guess I wasn't listening at school that day :)